enhancements/two-node-fencing/tnf.md

```diff
@@ -751,9 +785,26 @@ a close eye on when evaluating upgrade jobs for potential race conditions.
 
 ## Alternatives
 
-* MicroShift was considered as an alternative but it was ruled out because it does not support multi-node and has a very different experience than OpenShift which does not match the TNF initiative which is on getting the OpenShift experience on two nodes
+#### Active-Passive Single-Node OpenShift
```

Comment:
My major concerns with this section is that we're glossing over

1. What entity can ensure exactly one active cluster is chosen
2. What entity will route traffic to that cluster
3. How data is synchronised between the clusters, and in what direction
4. The need for bespoke configurations of (and possibly functional modifications to) Red Hat and partner workloads that expect to run multiple instances within a single configuration domain/cluster.

which make this alternative seem more realistic than it is.  
We should make it clear that this option faces all the same challenges as TNF, needs to solve them in the general case, plus introduces additional ones.